SPECS/python-click/python-click.spec

```diff
@@ -30,132 +20,214 @@ comes with good defaults out of the box.
 
 %package -n     python%{python3_pkgversion}-%{pypi_name}
 Summary:        %{summary}
-%{?python_provide:%python_provide python%{python3_pkgversion}-%{pypi_name}}
+
 BuildRequires:  python%{python3_pkgversion}-devel
-BuildRequires:  python%{python3_pkgversion}-setuptools
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python-wheel
+%if %{with_check}
+BuildRequires:  python%{python3_pkgversion}-pip
+%endif
 
+%global _description \
 %description -n python%{python3_pkgversion}-%{pypi_name} %{_description}
 
 %prep
 %autosetup -n %{pypi_name}-%{version} -p1
+# Use test dependencies without version locks
+sed -i 's|requirements/tests.txt|requirements/tests.in|' tox.ini
+# Unpin pytest version
+# https://github.com/pallets/click/commit/91b2cd67dd04b2843fd6c71e582876c826f5b78e
+sed -i 's|pytest<7|pytest|' requirements/tests.in
+
+%generate_buildrequires
+%pyproject_buildrequires -t
 
 %build
-%py3_build
+%pyproject_wheel
 
 %install
-%py3_install
+%pyproject_install
+%pyproject_save_files %{pypi_name}
 
 %check
-pip3 install pytest
-pip3 install .
-pytest -v tests
+%tox
 
-%files -n python%{python3_pkgversion}-%{pypi_name}
+%files -n python%{python3_pkgversion}-%{pypi_name} -f %pyproject_files
 %license LICENSE.rst
 %doc README.rst CHANGES.rst
-%{python3_sitelib}/%{pypi_name}/
-%{python3_sitelib}/%{pypi_name}-*.egg-info/
 
 %changelog
+* Tue Jun 21 2022 Sumedh Sharma <sumsharma@microsoft.com> - 8.0.4-3
+- Import from Fedora 36 (License: BSD)
```

Comment:
This should be MIT, and I believe this needs different wording to pass our checks- see the logs for details.

Reply:
This line should not be there. Removed this.